docs/summer-of-code-ideas/summer-of-code-reports/gsoc-2020-andreperezmaselco.md

```diff
@@ -0,0 +1,203 @@
+
+- [#3519](https://github.com/KhronosGroup/SPIRV-Tools/pull/3519): Support adding dead break from back-edge block
+
+- [#3521](https://github.com/KhronosGroup/SPIRV-Tools/pull/3521): Fix instruction insertion issue
+
+- [#3527](https://github.com/KhronosGroup/SPIRV-Tools/pull/3527): Implement the `OpMatrixTimesMatrix` linear algebra case
+
+- [#3587](https://github.com/KhronosGroup/SPIRV-Tools/pull/3587): Add condition to make functions livesafe
+
+- [#3589](https://github.com/KhronosGroup/SPIRV-Tools/pull/3589): Implement the `OpTranspose` linear algebra case
+
+- [#3610](https://github.com/KhronosGroup/SPIRV-Tools/pull/3610): Improve the code of the `Instruction` class
+
+- [#3617](https://github.com/KhronosGroup/SPIRV-Tools/pull/3617): Implement the `OpOuterProduct` linear algebra case
+
+- [#3654](https://github.com/KhronosGroup/SPIRV-Tools/pull/3654): Iterate over blocks in replace linear algebra pass
+
+- [#3664](https://github.com/KhronosGroup/SPIRV-Tools/pull/3664): Ignore specialization constants
+
+- [#3666](https://github.com/KhronosGroup/SPIRV-Tools/pull/3666): Fix in operand type assertion
+
+- [#3670](https://github.com/KhronosGroup/SPIRV-Tools/pull/3670): Check integer and float width capabilities
+
+- [#3672](https://github.com/KhronosGroup/SPIRV-Tools/pull/3672): Consider additional access chain instructions
+
+- [#3682](https://github.com/KhronosGroup/SPIRV-Tools/pull/3682): Add `spvOpcodeIsAccessChain` function
+
+- [#3694](https://github.com/KhronosGroup/SPIRV-Tools/pull/3694): Check header dominance when adding dead block
+
+- [#3710](https://github.com/KhronosGroup/SPIRV-Tools/pull/3710): Check termination instructions when donating modules
+
+- [#3711](https://github.com/KhronosGroup/SPIRV-Tools/pull/3711): Implement `opt::Function::HasEarlyReturn` function
+
+- [#3728](https://github.com/KhronosGroup/SPIRV-Tools/pull/3728): Add words instead of logical operands
+
+### Issues
+
+- [#3354](https://github.com/KhronosGroup/SPIRV-Tools/issues/3354): Replace linear algebra instruction with its mathematical definition
+
+- [#3484](https://github.com/KhronosGroup/SPIRV-Tools/issues/3484): Replace `OpSwitch` with blocks containing an `OpBranchConditional` instruction
+
+- [#3557](https://github.com/KhronosGroup/SPIRV-Tools/issues/3557): Split bit instructions into most fundamental operations
+
+- [#3588](https://github.com/KhronosGroup/SPIRV-Tools/issues/3588): Dynamically extract and insert vector components
+
+- [#3646](https://github.com/KhronosGroup/SPIRV-Tools/issues/3646): Replace decimal arithmetic with binary arithmetic
+
+## Conclusion
+It was great to participate in the *Google Summer of Code 2020* program. I was able to learn more about computer graphics, improve code writing and project collaboration. Finally, I would like to thank my mentors, Alastair Donaldson and Paul Thomson, for the opportunity.
+
+## References
```

Comment:
It's nice that you have references!

You should refer to the references earlier in the text.  E.g. you could refer to [1] if you mention the glsl-fuzz tool that was the precursor to spirv-fuzz.  (You might then also want to mention our paper "Automated testing for Graphics Shader Compilers" that has Paul as an author.)  I suppose you wanted to reference Abel and Jiradet's reports because you were inspired by their structure; in that case perhaps you should do it in a short note at the end of the conclusion.

In general when writing it's not good style to have items in a bibliography if you don't refer to those items from the main document.

Reply:
About the reports of Abel and Jiradet, I would like to leave them as references since I got in touch with the concepts I learned via their reports first.